neo/Persistence/Snapshot.cs

```diff
@@ -245,7 +245,7 @@ public IEnumerable<ECPoint> GetValidators(IEnumerable<Transaction> others)
             }
             int count = (int)snapshot.ValidatorsCount.Get().Votes.Select((p, i) => new
             {
-                Count = i,
+                Count = i + 1,
```

Comment:
Why this change?

Reply:
Because we use `Votes.Length - 1` as `index` when add votes.
https://github.com/neo-project/neo/blob/75df7cdaeffc84cd116d907ceda11054be696af3/neo/Ledger/Blockchain.cs#L481
https://github.com/neo-project/neo/blob/75df7cdaeffc84cd116d907ceda11054be696af3/neo/Ledger/Blockchain.cs#L509

And as @superboyiii described in #1799. Only 6 nodes became validators after vote 7 validators.